tests/post/features/authentication.feature

```diff
@@ -0,0 +1,34 @@
+@post @local @ci @authentication
+Feature: Authentication is up and running
+    Scenario: List Pods
+        Given the Kubernetes API is available
+        Then the 'pods' list should not be empty in the 'metalk8s-auth' namespace
+
+    Scenario: Expected Pods
+        Given the Kubernetes API is available
+        And pods with label 'release=nginx-ingress-control-plane' are 'Ready'
+        Then we have 2 running pod labeled 'app.kubernetes.io/name=dex' in namespace 'metalk8s-auth' on node 'bootstrap'
+
+    Scenario: Reach the OpenID Config
+        Given the Kubernetes API is available
+        And the control-plane Ingress path '/oidc' is available
```

Comment:
Would make sense to also use it in other scenarii that rely on it (Access HTTPS service, Login to Dex using incorrect email, Login to Dex using correct email and password)